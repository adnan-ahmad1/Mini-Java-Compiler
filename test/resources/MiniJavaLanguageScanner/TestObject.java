public class Square extends Shape {
    square.method(1, 10, 100);

    int[] i;
    int j;

    System.out.println(dog69);
}