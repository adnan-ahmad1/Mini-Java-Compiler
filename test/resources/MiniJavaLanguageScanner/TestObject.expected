PUBLIC CLASS ID(Square) EXTENDS ID(Shape) LCURLY ID(square) PERIOD ID(method) LPAREN INTEGER(1) COMMA INTEGER(10) COMMA INTEGER(100) RPAREN SEMICOLON INT LBRACKET RBRACKET ID(i) SEMICOLON INT ID(j) SEMICOLON PRINTLN LPAREN ID(dog69) RPAREN SEMICOLON RCURLY